src/components/Communication/Layout/Layout.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Flex, FlexItem, Button, Text } from "@fluentui/react-northstar";
+import { Illustration } from "../Illustration";
+
+export interface ILayout {
+  title?: string;
+  desc?: string;
+  image?: React.ReactNode;
+  actions?: ILayoutActions;
+}
+
+export interface ILayoutActions {
+  primary?: ILayoutAction;
+  secondary?: ILayoutAction;
+  tertiary?: ILayoutAction;
+}
+
+export interface ILayoutAction {
+  label: string;
+  action: () => void;
+}
+
+export const Layout = ({ title, desc, image, actions }: ILayout) => (
+  <Flex
+    hAlign="center"
+    gap="gap.large"
+    column
+    style={{
```

Comment:
Should we pull this out and put in separate css file like you do with Illustration.css?

Reply:
I don't see a reason to separate file for a few rows of the code:
```
      width: "100%",
      maxWidth: "33.5rem",
      margin: "3rem 1.25rem",
```

Just following current code style. 
We are declaring style props into a separate file in the case when we can't pass style properties into style attribute or trying to manipulate with elements that will be appended in the DOM in the future